### DISCUSSION TITLE
enable auto scroll on playing

### DISCUSSION THREAD

**Discussion by nadia-am** [CoderLine/alphaTab#623] [Q&A] [answered]
I used this library in my wordpress site, the auto scroll on notes doesnt work here. and when playing music i should scroll my self.
here is the link:
https://broken-guitar.ir/shop/gole-goldoone-man-notes-tab-simin-ghanem/
how can i fix this?
I really appriciate in advance

**Comment by nadia-am** [comment, marked as answer]
I fix the problem by addding `scrollElement: wrapper.querySelector('.at-viewport')` to the setting

**Comment by Danielku15** [comment]
To get fast feedback, you might want to rather share some minimal example specific to alphaTab than your whole project. I hope you can understand that people might be reluctant to give personal development support for your whole website and what you've developed. Giving feedback what you've done, what issue you're facing, what you've tried so far etc. are crucial. Otherwise people have to study everything you've developed and give feedback based on that.
